flang/lib/Lower/OpenMP/ClauseProcessor.cpp

```diff
@@ -394,6 +394,29 @@ bool ClauseProcessor::processNumThreads(
   return false;
 }
 
+bool ClauseProcessor::processOrder(mlir::omp::OrderClauseOps &result) const {
+  if (auto *clause = findUniqueClause<omp::clause::Order>()) {
```

Comment:
Nit: If you move the `using` statement above, you can simplify this statement as well.
```suggestion
  if (auto *clause = findUniqueClause<Order>()) {
```